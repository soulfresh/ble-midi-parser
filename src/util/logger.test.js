/* eslint-env jest */
/* eslint no-console: 0 */
test('log(...args)', () => {
  const consoleLog = console.log;
  console.log = jest.fn();
  const log = require('./logger').default;
  log();
  log('hi');
  log('hi', 'everybody');
  expect(console.log.mock).toMatchSnapshot();
  console.log = consoleLog;
});
