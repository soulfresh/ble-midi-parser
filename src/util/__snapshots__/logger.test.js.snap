// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`log(...args) 1`] = `
Object {
  "calls": Array [
    Array [],
    Array [
      "hi",
    ],
    Array [
      "hi",
      "everybody",
    ],
  ],
  "instances": Array [
    BufferedConsole {
      "_buffer": Array [],
      "assert": [Function],
      "clear": [Function],
      "count": [Function],
      "countReset": [Function],
      "debug": [Function],
      "dir": [Function],
      "dirxml": [Function],
      "error": [Function],
      "group": [Function],
      "groupCollapsed": [Function],
      "groupEnd": [Function],
      "info": [Function],
      "log": [Function],
      "time": [Function],
      "timeEnd": [Function],
      "trace": [Function],
      "warn": [Function],
      Symbol(counts): Map {},
      Symbol(groupIndent): "",
    },
    BufferedConsole {
      "_buffer": Array [],
      "assert": [Function],
      "clear": [Function],
      "count": [Function],
      "countReset": [Function],
      "debug": [Function],
      "dir": [Function],
      "dirxml": [Function],
      "error": [Function],
      "group": [Function],
      "groupCollapsed": [Function],
      "groupEnd": [Function],
      "info": [Function],
      "log": [Function],
      "time": [Function],
      "timeEnd": [Function],
      "trace": [Function],
      "warn": [Function],
      Symbol(counts): Map {},
      Symbol(groupIndent): "",
    },
    BufferedConsole {
      "_buffer": Array [],
      "assert": [Function],
      "clear": [Function],
      "count": [Function],
      "countReset": [Function],
      "debug": [Function],
      "dir": [Function],
      "dirxml": [Function],
      "error": [Function],
      "group": [Function],
      "groupCollapsed": [Function],
      "groupEnd": [Function],
      "info": [Function],
      "log": [Function],
      "time": [Function],
      "timeEnd": [Function],
      "trace": [Function],
      "warn": [Function],
      Symbol(counts): Map {},
      Symbol(groupIndent): "",
    },
  ],
}
`;
